docs/content/doc/usage/authentication.zh-cn.md

```diff
@@ -17,6 +17,264 @@ menu:
 
 # 认证
 
+{{< toc >}}
```

Comment:
This line is unnecessary now because we are not using hugo yet.

Reply:
ok